mavis/test/models/import_records.py

```diff
@@ -186,40 +165,38 @@ def import_vaccination_records(
         _input_file_path, _output_file_path = self.test_data.get_file_paths(
             file_paths=file_paths, session_id=session_id
         )
+        self.click_import_records()
+        self.select_vaccination_records()
+        self.click_continue()
+        self._upload_and_verify_output(_input_file_path, _output_file_path)
+
         if verify_on_children_page:
             _cl = self.test_data.create_child_list_from_file(
                 file_path=_input_file_path, file_type=file_type
             )
-        self.po.act(locator=self.LNK_IMPORT_RECORDS, action=actions.CLICK_LINK)
-        self.po.act(
-            locator=self.RDO_VACCINATION_RECORDS, action=actions.RADIO_BUTTON_SELECT
-        )
-        self.po.act(locator=self.BTN_CONTINUE, action=actions.CLICK_BUTTON)
-        self.po.act(
-            locator=self.LBL_VACCINATION_RECORDS,
-            action=actions.SELECT_FILE,
-            value=_input_file_path,
-        )
+            self.children_page.verify_child_has_been_uploaded(child_list=_cl)
+
+    def _upload_and_verify_output(self, _input_file_path, _output_file_path):
+        self.set_input_file(file_path=_input_file_path)
         self._record_upload_time()
-        self.po.act(locator=self.BTN_CONTINUE, action=actions.CLICK_BUTTON)
+        self.click_continue()
 
         if self.is_processing_in_background():
             self._click_uploaded_file_datetime()
             self.wait_for_processed()
 
         self._verify_upload_output(file_path=_output_file_path)
-        if verify_on_children_page:
-            self.children_page.verify_child_has_been_uploaded(child_list=_cl)
 
     def _record_upload_time(self):
         self.upload_time = datetime.now()
 
+    @step("Click on {0}")
```

Comment:
The `{0}` refers to the arguments passed in to the method, but in this case there are no arguments passed in, so it would appear in the report as `Click on`.